wp-multi-network/includes/classes/class-wp-ms-networks-api.php

```diff
@@ -0,0 +1,859 @@
+<?php
+/**
+ * REST API: WP_REST_Networks_Controller class
+ *
+ * @package WPMN
+ * @subpackage REST_API
+ * @since 2.2.0
+ */
+
+/**
+ * Core controller used to access networks via the REST API.
+ *
+ * @since 2.2.0
+ *
+ * @see WP_REST_Controller
+ */
+class WP_REST_Networks_Controller extends WP_REST_Controller {
```

Comment:
This should be called `WP_MS_REST_Networks_Controller`, similarly prefixed as the other classes.